gwnr/waveform/enigma_utils.py

```diff
@@ -211,18 +238,21 @@ def get_imr_enigma_modes(
         False,
     )
     t, x, e, l, phi, phidot, r, rdot = retval[:8]
-    t.data.data *= lal.MTSUN_SI
-    time_delta = time.time() - itime
+    t.data.data *= (
+        mass1 + mass2
+    ) * lal.MTSUN_SI  # Time from geometrized units to seconds
+
+    time_delta = time.perf_counter() - itime
     if verbose:
-        print("Orbital evolution took: {} seconds".format(time_delta))
+        print(f"Orbital evolution took: {time_delta} seconds")
```

Comment:
Do we need to create `time_delta` really? If the code is run in production mode with `verbose=False`, we can save this variable's creation by replacing:

--> `print(f"Orbital evolution took: {time.perf_counter() - itime} seconds")`

Reply:
You're right. I'll make this change.